org.eclipse.acute/src/org/eclipse/acute/AcutePreferencePage.java

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *  Lucas Bullen   (Red Hat Inc.) - Initial implementation
+ *******************************************************************************/
+package org.eclipse.acute;
+
+import static org.eclipse.swt.events.SelectionListener.widgetSelectedAdapter;
+
+import java.io.File;
+
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.jface.preference.PreferencePage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.FileDialog;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+
+public class AcutePreferencePage extends PreferencePage implements IWorkbenchPreferencePage {
+	public static String PAGE_ID = "org.eclipse.acute.preferencePage";
+	private IPreferenceStore store;
+
+	private Text explicitDotnetPathText;
+
+	@Override
+	public void init(IWorkbench workbench) {
+		store = doGetPreferenceStore();
+	}
+
+	@Override
+	protected Control createContents(Composite parent) {
+		Composite container = new Composite(parent, SWT.NULL);
+		container.setLayout(new GridLayout(2, false));
+
+		Label introLabel = new Label(container, SWT.WRAP);
+		introLabel.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 2, 1));
+		introLabel.setText("General aCute Settings:");
```

Comment:
Please avoid `aCute` in labels.
Moreover, I don't think we need an intro here, if we name the page .NET Core, I don't see how this line is helping.